dev/tools/dartdoc.dart

```diff
@@ -137,6 +137,14 @@ Future<void> main(List<String> arguments) async {
     '--header', 'snippets.html',
     '--header', 'opensearch.html',
     '--footer-text', 'lib/footer.html',
+    '--allow-warnings-in-packages', <String>[
+      'Flutter',
```

Comment:
Just curious: What uses this package? Seems like a mistake to name it "Flutter" instead of "flutter".

Reply:
This is the wrapper package used only for documentation, rooted at dev/docs.  Dartdoc doesn't (yet?) have a concept of a package consisting only of other packages, so there's been a stub package to import all the ones contained in Flutter since dartdoc was added to Flutter, AFAIK.